frontend/app/assets/stylesheets/spree/frontend/views/spree/shared/product.scss

```diff
@@ -5,13 +5,15 @@
     font-weight: 500;
     color: theme-color('dark');
     font-size: $font-size;
+    height: 1.5 * $font-size;
     line-height: $font-size;
     margin-top: calc-spacer(2);
 
     @include media-breakpoint-up(md) {
       $font-size: font-px-to-rem(18px);
 
       font-size: $font-size;
+      height: 1.5 * $font-size;
```

Comment:
This property isn't necessary because we set it outside of the media query.

Reply:
@piotrleniec-spark the font-size variable has been changed, so the property based on this has to be changed too.